src/Core/Code/InstructionTransformer.cs

```diff
@@ -249,15 +249,15 @@ public virtual Expression VisitProcedureConstant(ProcedureConstant pc)
 			return pc;
 		}
 
-		public virtual Expression VisitSegmentedAccess(SegmentedAccess access)
-		{
-			access.BasePointer = access.BasePointer.Accept(this);
-			access.EffectiveAddress = access.EffectiveAddress.Accept(this);
-			access.MemoryId = (MemoryIdentifier) access.MemoryId.Accept(this);
-			return access;
-		}
+        public virtual Expression VisitSegmentedAccess(SegmentedAccess access)
+        {
+            access.BasePointer = access.BasePointer.Accept(this);
+            access.EffectiveAddress = access.EffectiveAddress.Accept(this);
+            access.MemoryId = (MemoryIdentifier)access.MemoryId.Accept(this);
+            return access;
+        }
 
-		public virtual Expression VisitScopeResolution(ScopeResolution scope)
+        public virtual Expression VisitScopeResolution(ScopeResolution scope)
```

Comment:
@chostelet please do such minor style changes in separate commit(s) in branch(es) starting from `master` and then send PR(s). It makes easier to review this PR(changes will become smaller)